tests/cross/src/test/scala/tests/pc/CompletionArgSuite.scala

```diff
@@ -212,7 +252,12 @@ class CompletionArgSuite extends BaseCompletionSuite {
         |""".stripMargin,
     """|isResourceFile = : Boolean
        |isResourceFile = isLargeBanana : Boolean
-       |""".stripMargin
+       |""".stripMargin,
+    compat = Map(
```

Comment:
If the test case is ignored we can remove the compat section.

Reply:
This is for `arg14` not for `arg9`